### PR TITLE
Add stored values data route to Data Views documentation

### PR DESCRIPTION
**Pull request by emonino**

### CODE REVIEW

**Review by osimarkrogers — CHANGES_REQUESTED** [review]

File: content/api-reference/data-views/data-views-api-overview/data-api.md [hunk 1]
```diff
@@ -5,8 +5,8 @@ uid: DataViewsDataAPI
 # Data API
 The Data API allows users to [retrieve data](xref:DataViewsQuickStartGetData) for a specified data view.  This API is one portion of the [data views API](xref:DataViewsAPIOverview).
 
-## `Get Data View Data`
-Get data for the provided index parameters with paging. See [documentation on paging](xref:DataViewsQuickStartGetData#paging) for further information.
+## `Get Data View Interpolated Data`
+Get interpolated data for the provided index parameters with paging. See [documentation on paging](xref:DataViewsQuickStartGetData#paging) for further information.
```
[the remark this answers is not in the document]
Comment (emonino) [reply content/api-reference/data-views/data-views-api-overview/data-api.md]: Fixed

File: content/api-reference/data-views/data-views-api-overview/data-api.md [hunk 2]
```diff
@@ -230,3 +230,225 @@ Time,Temperature,Flowrate,Volume
             CacheBehavior cache = CacheBehavior.Refresh,
             CancellationToken cancellationToken = default);
 ```
+
+## `Get Data View Stored Data`
+Get stored data for the provided index parameters with paging. See [documentation on paging](xref:DataViewsQuickStartGetData#paging) for further information.
```
[the remark this answers is not in the document]
Comment (emonino) [reply content/api-reference/data-views/data-views-api-overview/data-api.md]: Fixed

File: content/api-reference/data-views/data-views-api-overview/preview-data-api.md [hunk 3]
```diff
@@ -270,3 +270,265 @@ Time,Temperature,Flowrate,Volume
             CacheBehavior cache = CacheBehavior.Refresh,
             CancellationToken cancellationToken = default);
 ```
+
+## `Get Data View Stored Data`
+Get stored data for the provided data view and index parameters with paging. See [documentation on paging](xref:DataViewsQuickStartGetData#paging) for further information.
```
[the remark this answers is not in the document]
Comment (emonino) [reply content/api-reference/data-views/data-views-api-overview/preview-data-api.md]: Fixed

File: content/api-reference/data-views/data-views-api-overview/preview-data-api.md [hunk 4]
```diff
@@ -270,3 +270,265 @@ Time,Temperature,Flowrate,Volume
...
+Successful (200 OK) responses include:
+
+| Header | Description |
+|--|--|
+| Link | Hyperlinks to the first page and next page of data as applicable. Absence of the next link indicates that there is no additional data to be retrieved. |
+| Next-Page | Hyperlink to the next page of data |
+| First-Page | Hyperlink to the first page of data |
+
+#### Example response body in default format
+An array of json values. Each json property corresponds to a field mapping. Property names are the field mapping `.Id`s.
```
[the remark this answers is not in the document]
Comment (emonino) [reply content/api-reference/data-views/data-views-api-overview/preview-data-api.md]: Fixed

File: content/api-reference/data-views/get-data-view-data/quick-start-get-data-view-data.md [hunk 5]
```diff
@@ -23,14 +23,14 @@ Three parameters control the range and granularity of data returned:
 |--|--|--|--|
 | Start index | `startIndex` | `DefaultStartIndex` | The inclusive start boundary of the data view data
 | End index | `endIndex` | `DefaultEndIndex` | The inclusive end boundary of the data view data
-| Interval | `interval` | `DefaultInterval` | The interval between index values
+| Interval | `interval` | `DefaultInterval` | The interval between index values. Only applies for interpolated retrieval mode.
```
[the remark this answers is not in the document]
Comment (emonino) [reply content/api-reference/data-views/get-data-view-data/quick-start-get-data-view-data.md]: Fixed

File: content/external-references/dataviews-summaries.yaml [hunk 6]
```diff
@@ -85,15 +85,25 @@ dataviews-acl-update:
     Updates the default [`AccessControlList`](https://ocs-docs.osisoft.com/Content_Portal/Documentation/Access_Control.html#access-control-lists) for the DataViews collection.
 
 # method summaries: requesting data
-data-get:
+data-interpolated-get:
   type: markdown
   value: |
-    Gets data for the provided index parameters with paging. See [documentation on paging](https://ocs-docs.osisoft.com/Content_Portal/Documentation/DataViews/GetDataViewData/Quick_Start_Get_Data_View_Data.html#paging) for further information.
+    Gets interpolated data for the provided index parameters with paging. See [documentation on paging](https://ocs-docs.osisoft.com/Content_Portal/Documentation/DataViews/GetDataViewData/Quick_Start_Get_Data_View_Data.html#paging) for further information.
```
[the remark this answers is not in the document]
Comment (emonino) [reply content/external-references/dataviews-summaries.yaml]: Fixed

File: content/api-reference/data-views/data-views-api-overview/data-api.md [hunk 7]
```diff
@@ -230,3 +230,225 @@ Time,Temperature,Flowrate,Volume
...
+`[optional] int count`
+The requested page size. The maximum is 250,000. If the parameter is not provided, [an optimal page size will be calculated](xref:DataViewsQuickStartGetData#page-size).
+
+### Response
+The response includes a status code and, in most cases, a body.
+
+| Status code | Body Type | Description |
+|--|--|--|
+| 200 OK                    | data in the requested format  | Successfully retrieved data.  |
+| 400 Bad Request           | error | The request could not be understood by the server due to malformed syntax.
```
[the remark this answers is not in the document]
Comment (emonino) [reply content/api-reference/data-views/data-views-api-overview/data-api.md:281]: Does the lower case even apply for status code responses? We've always used all uppercase and that seems to be the convention elsewhere in the OCS documentation. Here is an example from Assets - https://docs.osisoft.com/bundle/ocs/page/api-reference/assets/assets-api.html.

File: content/api-reference/data-views/data-views-api-overview/data-api.md [hunk 8]
```diff
@@ -230,3 +230,225 @@ Time,Temperature,Flowrate,Volume
...
+| 403 Forbidden             | error | User is not authorized for this operation.
+| 404 Not Found             | error | The specified data view identifier is not found.
```
[the remark this answers is not in the document]
Comment (emonino) [reply content/api-reference/data-views/data-views-api-overview/data-api.md:283]: Skipping for now. Will fix if we decide that status codes should be lower case.

File: content/api-reference/data-views/data-views-api-overview/data-api.md [hunk 9]
```diff
@@ -230,3 +230,225 @@ Time,Temperature,Flowrate,Volume
...
+| 500 Internal Server Error | error | An error occurred while processing the request. See the response body for details |
```
[the remark this answers is not in the document]
Comment (emonino) [reply content/api-reference/data-views/data-views-api-overview/data-api.md:284]: Skipping for now. Will fix if we decide that status codes should be lower case.

File: content/api-reference/data-views/data-views-api-overview/preview-data-api.md [hunk 10]
```diff
@@ -270,3 +270,265 @@ Time,Temperature,Flowrate,Volume
...
+}
+```
+
+### Response
+The response includes a status code and, in most cases, a body.
+
+| Status code | Body Type | Description |
+|--|--|--|
+| 200 OK                    | data in the requested format  | Successfully retrieved data.  |
+| 400 Bad Request           | error | The data view or the query parameters are not valid. See the response body for details. |
```
[the remark this answers is not in the document]
Comment (emonino) [reply content/api-reference/data-views/data-views-api-overview/preview-data-api.md:360]: Skipping for now. Will fix if we decide that status codes should be lower case.

File: content/api-reference/data-views/data-views-api-overview/preview-data-api.md [hunk 11]
```diff
@@ -270,3 +270,265 @@ Time,Temperature,Flowrate,Volume
...
+| 403 Forbidden             | error | User is not authorized to create a data view.
+| 500 Internal Server Error | error | An error occurred while processing the request. See the response body for details |
```
[the remark this answers is not in the document]
Comment (emonino) [reply content/api-reference/data-views/data-views-api-overview/preview-data-api.md:362]: Skipping for now. Will fix if we decide that status codes should be lower case.

File: content/api-reference/data-views/data-views-api-overview/data-api.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (emonino) [reply content/api-reference/data-views/data-views-api-overview/data-api.md]: I fixed the entire data views API documentation to use lower case lettering for the second word in a status code

**Review by osimarkrogers — APPROVED** [review]

**Pull request merged**